### PR TITLE
Add HA secrets

### PR DESCRIPTION
**Pull request by Josipmrden**
Add environment flags for USER and PASSWORD for HA Helm Charts

### CODE REVIEW

File: charts/memgraph-high-availability/values.yaml [hunk 1]
```diff
@@ -62,6 +62,12 @@ sysctlInitContainer:
   enabled: true
   maxMapCount: 262144
 
+secrets:
```
Comment (antejavor) [inline charts/memgraph-high-availability/values.yaml:65]: This needs to end up in the README describing the property: https://github.com/memgraph/helm-charts/blob/main/charts/memgraph-high-availability/README.md#configuration-options

Comment (antejavor) [reply charts/memgraph-high-availability/values.yaml:65]: This also needs to be added to the docs page, [something like this on a HA part of the page](https://memgraph.com/docs/getting-started/install-memgraph/kubernetes#secrets)  also [here](https://memgraph.com/docs/getting-started/install-memgraph/kubernetes#configuration-options-1)

Comment (antejavor) [reply charts/memgraph-high-availability/values.yaml:65]: Can you just confirm you have been able to setup the secrets inside the cluster and connect to Memgraph instances with auth? 

From the code perspective it looks ok, but our CI/CD is currently not operational for HA so I want to be sure this works.

**Review by antejavor — CHANGES_REQUESTED** [review]

File: charts/memgraph-high-availability/values.yaml (hunk 1 above)
Comment (Josipmrden) [reply charts/memgraph-high-availability/values.yaml:65]: Yes, I have now tested and it works.

Comment (antejavor) [reply charts/memgraph-high-availability/values.yaml:65]: This is resolved.

**Review by antejavor — APPROVED** [review]
LGTM.

**Review by as51340 — COMMENTED** [review]
secrets should be under memgraph. Doesn't make sense that we have values.secrets and values.memgraph.env. Everything should be under same env part of values.yaml.

@Josipmrden or @antejavor please make a PR and fix this

**Comment by antejavor** [comment]
Good thinking, @as51340,  I have [opened a related issue](https://github.com/memgraph/helm-charts/issues/96) for this comment, since we released this already without the docs in a fast-paced matter because of the nature of this request. 

I will fix it before the next release, since I guess not that critical. Wdyt @as51340?

**Pull request merged**